### ISSUE TITLE
[HW] Joseph Wang

### ISSUE THREAD

**Issue by EmeraldEntities** [datastaxdevs/workshop-graphql-netflix#207]
**Name:** Joseph Wang
**Email:** wang.joseph1516@gmail.com
**Linkedin Profile:** https://www.linkedin.com/in/joseph-wang1516/

Attach the homework screenshots below:
-----------------------------------------

![image](https://user-images.githubusercontent.com/44278515/124162694-be5cad80-da6c-11eb-92c8-df284cd4e4d3.png)

Proof of movie upload:

![image](https://user-images.githubusercontent.com/44278515/124162846-f06e0f80-da6c-11eb-88a1-e636f2ddfd22.png)

Site: https://friendly-golick-261ef4.netlify.app/

**Comment by EmeraldEntities** [comment]
PS. yes I did change the react code to only show action movies
I hope that is okay :)

**Comment by SonicDMG** [comment]
Yes, no worries at all. You did everything we asked per the homework. Awesome job. Here's your badge -> https://api.badgr.io/public/assertions/0KSk0AGMRHa9f_-P4NRDvQ?identity__email=wang.joseph1516%40gmail.com
